src/commands/dump.js

```diff
@@ -143,6 +163,25 @@ module.exports = async options => {
     secretAccessKey: secret_access_key
   }
   const cos = new COS.S3(cosConfig)
+
+  if (ops.annotations){
+    //only download pictures and annotations
+    spinner.message = `Exporting annotations from ${bucket}...`
+    await downloadAnnotations(cos, bucket, 'exported_buckets')
+    
+    if(fs.existsSync(`exported_buckets/${bucket}/_annotations.json`)){
+      fs.renameSync(`exported_buckets/${bucket}/_annotations.json`,`exported_buckets/${bucket}/annotations.json`)
```

Comment:
Is there a reason you rename this from `_annotations.json` to `annotations.json`?  This will break `cacli bootstrap`